app/bin/run-ecs-task/network_config.json.tpl

```diff
@@ -0,0 +1,7 @@
+{
+  "awsvpcConfiguration": {
+    "assignPublicIp": "ENABLED",
+    "securityGroups": [],
+    "subnets": []
```

Comment:
Is this where the defaults should be set once we have them?

Reply:
No, anything written in securityGroups or subnets gets 100% overriden by the script as written (either based on CLI inputs or whats found in AWS). Defaults would go in the commands that query AWS with whatever security group name(s) we default to + the VPC you want to run in to get all of its subnets.

Reply:
To make this more clear, I'm setting the "default" value that'll be overriden to something saying to go look at the readme.